_docs-sources/guides/reference-architecture/configuration-guide/index.md

```diff
@@ -18,220 +18,291 @@ Caveat: at this time, the Reference Architecture does not configure or manage th
 
 :::
 
-## Clone this repository
+## 1. Create an infrastructure-live repository
 
-The very first step is to clone this repository to your local machine. You must have Git installed on your machine. Refer to [these instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) to install Git on your platform.
+1. Create a new repository in your VCS platform. We recommend naming it _infrastructure-live_.
+1. Keep this repo handy, as you'll be prompted for the following information in a subsequent step:
+    - HTTPS URL (e.g. `https://github.com/gruntwork-io/infrastructure-live`)
+    - SSH URL (e.g. `git@github.com:gruntwork-io/infrastructure-live.git`)
+    - Default branch (e.g. `main` or `master`)
 
-1. Clone the repository.
+## 2. Set up the machine user
 
-  ```bash
-  git clone git@github.com:gruntwork-clients/<this-repo>.git
-  ```
+The next step is to configure the Machine User Personal Access Token(s)
 
-1. Check out a new branch with your changes:
+If you are using GitHub to host your `infrastructure-live` repository, you will only need the one 
+personal access token as the permissions will allow access to both your `infrastructure-live` repo and 
+Gruntwork's private repositories.
 
-  ```bash
-  git checkout -b ref-arch-form
-  ```
+If you are using GitLab or Bitbucket to host your `infrastructure-live` repository, you will need a 
+Personal Access Token for your respective VCS in addition to a GitHub PAT for access to the 
+private Gruntwork GitHub repositories. Note that at this time GitHub is the only supported VCS for 
+Reference Architecture deployments.
 
-## Create and configure AWS Accounts
+First we will create a GitHub Personal Access Token:
```

Comment:
Should we define the shorthand here, rather than in the first step below?

```suggestion
First we will create a GitHub Personal Access Token (PAT):
```